src/schema.mjs

```diff
@@ -102,6 +102,11 @@ export const configSchema = {
       description: "Base directory for user SSH keys. Supports templating with %u (username) and %U (uid)",
       default: "/home/%u/.ssh",
     },
+    directories: {
```

Comment:
```suggestion
    managed_user_directories: {
```

Reply:
Just a more specific name, in case we have other types of directories in the future (we prob will have some kind of group directory implementation).